modules/concertBidAdapter.js

```diff
@@ -203,9 +204,8 @@ function hasOptedOutOfPersonalization() {
  * @param {BidderRequest} bidderRequest Object which contains any data consent signals
  */
 function consentAllowsPpid(bidderRequest) {
-  /* NOTE: We cannot easily test GDPR consent, without the
+  /* NOTE: We can't easily test GDPR consent, without the
```

Comment:
what does this comment mean? why can't you look at the tcf consent string without this npm module?

Reply:
We can _view_ the consent string, yes, but if _parsing_ it is non-trivial enough that there exists an official, specialised, and sanctioned `npm` module to do it, it does not seem appropriate that we would try to add that functionality ourselves to our bid adapter; especially since we do that exact step (using `consent-string`) in our bid server.

(For the record, this change only exists in this PR because it was a `NOP` commit, to try to activate the CircleCI on a fresh push.)

Reply:
we provide a utility function, eg https://github.com/prebid/Prebid.js/blob/0be4560e182e46c121c283f763cdef74b5ff9db9/modules/appnexusBidAdapter.js#L559

Reply:
I appreciate this, @patmmccann , thank you! (I don't think those functions existed when we first wrote our adapter, so it's good to know that so we can update it.) I've updated our use of consent strings, as per these comments, and will ask Chris if they can merge these changes to this branch (which I can't do). 🙌